.github/workflows/ci.yml

```diff
@@ -61,6 +61,15 @@ jobs:
       - name: Test 1 - Run Init Test
         run: |
           ./tests/testinit
+      - name: Test 2 - Compile Version Test
+        uses: suve/GHActions-FPC@v0.3.2
+        with:
+          source: tests/testversion.pas
+          flags: Fl/usr/local/lib
```

Comment:
This shouldn't be required.

Reply:
Why? - Without this part testversion.pas is not compiled, right?

Reply:
I mean strictly this line, where `-Fl/usr/local/lib` is passed.

Reply:
The reason why it is there, is that without this line the sdl2 library is not found. From my memory, it is related to a new way of storing libraries on big sur, they are in fact not some files stored in a folder anymore but in a more sophisticated indexed "library file". 

See. [https://github.com/PascalGameDevelopment/SDL2-for-Pascal/pull/36/commits/8f7b0106574bcb70e44f54e401af979c62fc1214](https://github.com/PascalGameDevelopment/SDL2-for-Pascal/pull/36/commits/8f7b0106574bcb70e44f54e401af979c62fc1214)

Reply:
Ah, sorry, I missed that this is part of the Mac job. I suppose that since `brew` is not the *official* package manager, the authors decided to put stuff in `/usr/local` over `/usr`.

Reply:
I see :)